src/tensors/linalg.jl

```diff
@@ -172,15 +193,11 @@ LinearAlgebra.isdiag(t::AbstractTensorMap) = all(LinearAlgebra.isdiag, values(bl
 # Copy, adjoint! and fill:
 function Base.copy!(tdst::AbstractTensorMap, tsrc::AbstractTensorMap)
     space(tdst) == space(tsrc) || throw(SpaceMismatch("$(space(tdst)) ≠ $(space(tsrc))"))
-    for c in blocksectors(tdst)
-        copy!(StridedView(block(tdst, c)), StridedView(block(tsrc, c)))
-    end
+    copy!(tdst.data, tsrc.data)
```

Comment:
This is not really an `AbstractTensorMap` implementation, but rather a `TensorMap` implementation, so I would tend to keep the previous definition and just specialize a version for TensorMap